### PR TITLE
Update ispmail_userctl.py

### PR DESCRIPTION
**Pull request by pragmatiker**
Thank you for writing this awesome tool for the community.

I think it should be possible to have an alias delivered to more than one Mailbox.
At least in postfix alias tables it is possible.

```
hr@domain.net judy@domain.net
hr@domain.net marc@domain.net
```

Then, when both alias and destination have been entered, check if the combination of alias AND destination already exists in the DB

### CODE REVIEW

File: ispmail_userctl.py [hunk 1]
```diff
@@ -810,18 +810,18 @@ def domain_add_alias_win(parent, window, top_title, domain):
...
-
     handle3 = SingleInput(parent, window, 'Add Alias (2/2)', top_title, 'Enter the new alias destination (supply the full address, with domain):', True)
     destination = handle3.run()
     if not destination or destination.find('@') == -1:
         handle4 = Note(parent, window, 'Add Alias Failed', top_title, 'Could not add new alias: invalid destination.')
         handle4.run()
         return
 
+    if any(alias.source == source + '@' + domain.name and alias.destination == destination for alias in db_get_aliases(domain)):
+        handle2 = Note(parent, window, 'Add Alias Failed', top_title, 'Could not add new alias: alias already exists.')
```
Comment (cgzones) [inline ispmail_userctl.py]: Please rename the handles, so their names are still in order.

**Review by cgzones — CHANGES_REQUESTED** [review]
Thanks for your patch; multiple destinations for the same alias is indeed a valid use case.

The alias deletion screen needs an update, since it currently only shows the source, which is now not unique anymore.

The listing screens (global and domain) can be updated to group destinations for the same source.

File: ispmail_userctl.py (hunk 1 above)
Comment (pragmatiker) [reply ispmail_userctl.py]: Soryy. I have really no usefull knowledge of ncurses yet. I will have to read up on how to do this right.

**Comment by cgzones** [comment]
Thanks for your contribution!

Applied via https://github.com/cgzones/ispmail-userctl/commit/2a6a95b261bcb0d30ddfa2a56c602140fb48e5e8 and https://github.com/cgzones/ispmail-userctl/commit/aca7a8961dc7ca7d6ac7acb5b80eb12f3bf9146f.

**Pull request closed without merging**